### ISSUE TITLE
Quit crash

### ISSUE THREAD

**Issue by ShadowOfNight** [TheLegendOfMataNui/game-issues#17]
Hitting quit on the main menu causes a crash.

**Comment by ShadowOfNight** [comment]
This is no longer an issue in the beta, so this is being closed.

**Comment by ShadowOfNight** [comment]
Managed to bypass the crash by fixing some issues with base_screenlist class, but there appears to be an additional crash taking place afterwards in GcShadow::Exit().  Further investigation is needed

**Comment by LennyTheBionicleLover** [comment]
It’s been a long time. Is this fixed yet?

**Comment by ShadowOfNight** [comment]
@LennyTheBionicleLover 

No, it has not been fixed since it's been a lower priority alpha issue.  I was posting an update in this thread because I did partially fix it and discovered another problem related to something we're looking into in the beta.  Hopefully we can get this resolved finally for the next patch.
